oclc.py

```diff
@@ -1353,6 +1351,32 @@ def process_item(self, identifier):
             )
         return identifier
 
+    def apply_viaf_to_contributor_data(self, metadata):
+        """Looks up VIAF information for contributors identified by OCLC
+
+        This is particularly crucial for contributors identified solely
+        by VIAF IDs (and no sort_name), as it raises errors later in the
+        process.
+        """
+        for contributor_data in metadata.contributors:
+            viaf_contributor_data = self.viaf.lookup_by_viaf(
+                contributor_data.viaf,
+                working_sort_name=contributor_data.sort_name,
+                working_display_name=contributor_data.display_name
+            )[0]
```

Comment:
Looking at the VIAF code I think it's quite possible lookup_by_viaf could return an empty list, which would cause a crash here.